wolox-books/src/app/services/auth.service.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+import { Observable, throwError } from 'rxjs';
+import { catchError, tap } from 'rxjs/operators';
+
+import { environment } from '../../environments/environment';
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+
+  readonly ROOT_URL = environment.BACKEND_URL;
+
+  constructor(
+    private http: HttpClient
+  ) { }
```

Comment:
No need to use that much of lines and space in the file.. write it all in 1 line

```suggestion
  constructor(private http: HttpClient) { }
```